src/lib.rs

```diff
@@ -174,7 +191,10 @@ fn json_get(ctx: &Context, args: Vec<String>) -> RedisResult {
             Err(_) => {
                 // path is optional -> no path found on the first loop we use root "$"
                 if first_loop {
-                    paths.push("$".to_owned());
+                    paths.push(PathBackward {
```

Comment:
I would add a `new` constructor to `PathBackward` instead of directly accessing the fields here.

Reply:
done